rubix_admin/installer.py

```diff
@@ -34,10 +36,23 @@ def install(cls, args):
         cls._rpm_install(args)
         cls._rubix_op(args)
 
+    @classmethod
+    def get_rpm_path(cls, args):
+        rpm_path = []
+        if args.rpm is not None:
+            rpm_path = args.rpm
+        else:
+            rpm_file_name = "qubole-rubix-" + args.rpm_version + ".noarch.rpm"
```

Comment:
nit: we can write to stdout that we are using default rubix rpm if arg is missing

Reply:
+1 to adding a log line